src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -1,11 +1,12 @@
 import * as React from 'react';
 import {View, Text, StyleSheet, ActivityIndicator} from 'react-native';
-
+import PropTypes from 'prop-types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `PropTypes` is added but it's not used for all props in the component. It's a good practice to define PropTypes for all props to catch bugs early and improve code readability.
```diff
- BottomSheetBio.propTypes = {
-   handleSave: PropTypes.func
- };
+ BottomSheetBio.propTypes = {
+   handleSave: PropTypes.func,
+   isOtherProfile: PropTypes.bool,
+   value: PropTypes.string,
+   error: PropTypes.string,
+   isLoadingUpdateBio: PropTypes.bool
+ };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->